kmultiversx/src/kmultiversx/kdist/mx-semantics/esdt.md

```diff
@@ -134,11 +134,12 @@ module ESDT
           </esdtData>)
           ...
         </account>
-      [priority(61)]
+      [priority(61), preserves-definedness]
+      // preserves-definedness: ACCT exists prior so the account map is defined
```

Comment:
I know that this PR is a draft, but I thought it does not hurt to point out that this comment is missing something (there are other similar comments in the PR): If TOKEN already exists in the map, then this does not preserve definedness, and this works only because there is a higher priority rule that makes sure that TOKEN is not in the map.

Reply:
Thanks for the suggestion - indeed this should be added to the comment(s) here. Will do so, and mark the PR ready for review.